.github/workflows/scripts/jira_helper.py

```diff
@@ -159,19 +183,23 @@ def _get_issue_type(self, labels) -> str:
         else:
             return 'Task'
 
-    def _get_gh_issue_labels(self):
+    def _get_gh_issue_labels(self, make_jira_compat: bool):
```

Comment:
`make_jira_compat` is always true? Just remove the option?

Reply:
👍